### ISSUE TITLE
httpagentparser.detect returns browser as AndroidBrowser instead of Firefox when using Firefox for Android.

### ISSUE THREAD

**Issue by SurenNihalani** [shon/httpagentparser#77]
```import httpagentparser
In [3]: httpagentparser.detect('Mozilla/5.0 (Android 6.0.1; Mobile; rv:63.0) Gecko/63.0 Firefox/63.0')
Out[3]:
{'bot': False,
 'browser': {'name': 'AndroidBrowser'},
 'dist': {'name': 'Android', 'version': '6.0.1'},
 'platform': {'name': 'Android', 'version': '6.0.1'}}```

**Comment by shon** [comment]
@SurenNihalani thanks for reporting. This is fixed and new version `1.8.2` is uploaded to pypi.

**Comment by SurenNihalani** [comment]
Thanks for your time @shon!
